Lib/sysconfig.py

```diff
@@ -1,8 +1,10 @@
 """Access to Python's configuration information."""
 
 import os
+import re
 import sys
 from os.path import pardir, realpath
+from distutils.errors import DistutilsPlatformError
```

Comment:
It would be nice not to import this, since what will it do once distutils is gone?

Reply:
It looks like it's only used for a couple of errors in new functions, so can probably make them return None and then wrap them in distutils.sysconfig.

Reply:
It'll need to be removed eventually, but it doesn't need to be done in this PR.

Reply:
I agree with encukou. I'd like to focus this PR only on sysconfig module as much as possible.